Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml

```
@@ -1,9 +1,9 @@
 #shuttle walls
 - type: entity
   id: WallAbductorDiagonal
-  name: shuttle wall
+  name: nanolaminate wall # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)